src/Components/RideProgressCard/Components/ProgressLine.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+import { Animated, View, StyleSheet, Easing } from "react-native";
+import Colors from "src/Themes/Colors";
+
+type Props = {
+  currentProgress?: number;
+};
+
+type State = {
+  progress: Animated.Value;
+};
+
+export class ProgressLine extends React.PureComponent<Props, State> {
```

Comment:
Why to use named export to export a component?

Reply:
And this component required a storybook too, to view it in isolation in different use-cases